dipy/core/tests/test_gradients.py

```diff
@@ -27,6 +30,79 @@ def test_unique_bvals_deprecated():
                       np.array([0, 800, 1400, 1401, 1405]))
 
 
+def test_read_bvals_bvecs(tmp_path: pathlib.Path):
```

Comment:
this test needs to be moved to `dipy.io.tests.test_gradients.py`.    See [here](https://github.com/dipy/dipy/blob/master/dipy/io/tests/test_io_gradients.py#L13)

You are currently in the `core` module instead of the `io` module.

I know, it can be confusing, but you are currently duplicating some tests. Sorry for that.

Reply:
I double checked and I think every case I wrote pretty much already existed so I removed this file and replaced it with the contents from master